src/main/java/com/adobe/commerce/cif/api/HealthApi.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.commerce.cif.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK;
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK_MESSAGE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE_MESSAGE;
+
+@Path("/_health")
```

Comment:
Why `/_health`? Why the `_`?

Reply:
I saw that with some services. They use that as convention to mark this endpoint as not part of the "functional" ones.

Reply:
Ok, makes sense, lets go for it.